lisp/ui/widgets/dbmeter.py

```diff
@@ -43,14 +44,23 @@ def __init__(
         self.dbClipping = clipping
         self.valueSmoothing = smoothing
         self.scale = scale
+        self.unit = unit
 
         self.backgroundColor = QColor(32, 32, 32)
         self.borderColor = QColor(80, 80, 80)
         self.clippingColor = QColor(220, 50, 50)
 
         self._currentSmoothing = self.valueSmoothing
+        self._markings = []
         self._pixmap = QPixmap()
 
+        font = self.font()
+        font.setPointSize(font.pointSize() - 2)
```

Comment:
I think we should use a fixed/monospaced font here, it should improve readability, and make the character size more consistent. In this case I've also noticed that's better to make it a bit smaller

```suggestion
        font = QFontDatabase.systemFont(QFontDatabase.FixedFont)
        font.setPointSize(font.pointSize() - 3)
```

Reply:
Done